### PR TITLE
Neutroswap Milestone 3

### PR DESCRIPTION
**Pull request by jonassunandar**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/eosnetworkfoundation/grant-milestones/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form](https://forms.gle/wLuAzXKa9qYrZQob9) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the EOS account name provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/eosnetworkfoundation/grant-framework/blob/master/docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/eosnetworkfoundation/grant-framework/pull/222 < please fill this in with the PR number of your application.

### CODE REVIEW

**Comment by nsjames** [comment]
I think there might be an error with the `.md` file here. It says it's for milestones 1 and 2, but this PR is for milestone 3. 

I also don't see the required deliverables listed there (the ones specified [here](https://github.com/eosnetworkfoundation/grant-framework/blob/main/applications/tokenomics_enhancement_neutroswap.md#milestone-3---mainnet-deployment--marketing))

**Comment by jonassunandar** [comment]
Hi @nsjames 
Sorry for pushing the wrong milestone 1 `.md`. Now it should be only for milestone 3

File: .DS_Store
Comment (wanderingbort) [inline .DS_Store:1]: please remove this file from the PR

File: deliveries/tokenomics_enhancement_neutroswap_milestone_3.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+| 0a.    | License | [LICENSE](https://github.com/neutroswap/neutroswap-v2-contracts/blob/main/LICENSE.md) |          |
+| 0b.    | Article | [Neutroswap V2 Introduction Thread](https://x.com/Neutroswap/status/1750911285938753575?s=20) | We published an informative threads on twitter, explaining the concept of V2 smart contracts and how they will be utilized within our project. |
+| 1.    | Smart Contract Deployment | [Mainnet Smart Contract List](https://github.com/neutroswap/neutroswap-v2-contracts?tab=readme-ov-file#mainnet-contracts) | We've deployed all contract necessary for V2 upgrade on EOS-EVM mainnet. |
+| 2.    | Smart Contract Code Verification | [Mainnet Smart Contract List](https://github.com/neutroswap/neutroswap-v2-contracts?tab=readme-ov-file#mainnet-contracts) | We've verified the smart contract codes on official EOS-EVM mainnet Explorer by providing ABIs, bytecodes, and contract addresses. |
+| 3.    | Graph Node Deployment	 | [Graph Index & Query Node](https://dbi.foundation/) | We've published a private graph-node instance for the EOS-EVM mainnet environment. |
```
Comment (wanderingbort) [inline deliveries/tokenomics_enhancement_neutroswap_milestone_3.md:38]: This link produces a 404 which I assume is due to it being "private" If that is intentional we can remove the link as the subgraph link below works just fine.

**Review by wanderingbort — CHANGES_REQUESTED** [review]

File: deliveries/tokenomics_enhancement_neutroswap_milestone_3.md (hunk 1 above)
Comment (jonassunandar) [reply deliveries/tokenomics_enhancement_neutroswap_milestone_3.md]: I've corrected the link to the subgraph query dashboard. My apologies for providing the incorrect link initially.

File: .DS_Store
Comment (jonassunandar) [reply .DS_Store:1]: I've removed the .DS_Store file

**Comment by nsjames** [comment]
Looks good. Swapped 1 EOS to USDT and it worked well (though with a lot of slippage)

**Review by nsjames — APPROVED** [review]

**Review by jonassunandar — COMMENTED** [review]
resolved broken link changes and removed .DS_STORE

**Comment by jonassunandar** [comment]
Hi @wanderingbort do you have any other question regarding the milestone? I'd love to fix it asap

**Pull request merged**